.tekton/developer-sandbox-ui-pull-request.yaml

```
@@ -27,7 +27,7 @@ spec:
   - name: image-expires-after
     value: 5d
   - name: dockerfile
-    value: ./Dockerfile
+    value: .gitmodules/Dockerfile
```

This value should be `build-tools`
```suggestion
    value: build-tools/Dockerfile
```